src/main/java/frc/lib/controllers/position/PositionControllerTalonFXElevator.java

```diff
@@ -179,9 +180,11 @@ public void periodic() {
     } else {
       Angle motorPosition = position.getValue().plus(positionOffset);
 
-      double feedbackVolts = feedback.calculate(motorPosition.in(Radians), setpointPosition.in(Radians));
       double feedforwardVolts = feedforward.calculate(setpointVelocity.in(RadiansPerSecond));
 
+      // If velocity setpoint is not 0, add some feedback voltage from PID controller
+      double feedbackVolts = (MathUtil.isNear(0.0, setpointVelocity.in(RotationsPerSecond), 0.001)) ? feedback.calculate(motorPosition.in(Radians), setpointPosition.in(Radians)) : 0.0;
```

Comment:
Probably isn't necessary since the velocity would be set to zero, but always good to avoid round-off. Also, this will probably removed once feedback can be smoothly introduced.